modules/dnn/test/test_darknet_importer.cpp

```diff
@@ -115,6 +116,42 @@ class Test_Darknet_layers : public DNNTestLayer
         net.setInput(inp);
         Mat out = net.forward();
         normAssert(out, ref, "", default_l1, default_lInf);
+
+        if (inp.size[0] == 1)  // test handling of batch size
```

Comment:
Can we add a boolean flag to skip this test? Looks like there is a problem with Myriad target and `testDarknetLayer("shortcut");`:

```
[ RUN      ] Test_Darknet_layers.shortcut/1, where GetParam() = DLIE/MYRIAD
/build/precommit_custom_linux/3.4/opencv/modules/dnn/test/test_darknet_importer.cpp:137: Failure
Expected equality of these values:
  0
  cv::norm(out2(ranges0), out2(ranges1), NORM_INF)
    Which is: 4.29004
Batch result is not equal: shortcut
Google Test trace:
/build/precommit_custom_linux/3.4/opencv/modules/dnn/test/test_darknet_importer.cpp:122: batch size 2
/build/precommit_custom_linux/3.4/opencv/modules/dnn/test/test_darknet_importer.cpp:102: shortcut
/build/precommit_custom_linux/3.4/opencv/modules/dnn/test/test_common.impl.hpp:66: Failure
Expected: (normL1) <= (l1), actual: 1.97884 vs 0.004
Google Test trace:
/build/precommit_custom_linux/3.4/opencv/modules/dnn/test/test_darknet_importer.cpp:122: batch size 2
/build/precommit_custom_linux/3.4/opencv/modules/dnn/test/test_darknet_importer.cpp:102: shortcut
/build/precommit_custom_linux/3.4/opencv/modules/dnn/test/test_common.impl.hpp:69: Failure
Expected: (normInf) <= (lInf), actual: 4.29028 vs 0.02
Google Test trace:
/build/precommit_custom_linux/3.4/opencv/modules/dnn/test/test_darknet_importer.cpp:122: batch size 2
/build/precommit_custom_linux/3.4/opencv/modules/dnn/test/test_darknet_importer.cpp:102: shortcut
[  FAILED  ] Test_Darknet_layers.shortcut/1, where GetParam() = DLIE/MYRIAD (8220 ms)
```